client/app/optionsets/landuse-form.js

```diff
@@ -0,0 +1,14 @@
+export const CEQR_TYPE = {
```

Comment:
hmm, small nitpick but.. conceptually from a CRM/business process perspective, CEQR_TYPE may not be scoped to the form. In CRM we set CEQR_TYPE on the project, (through the actions tab)  right?

Reply:
I just based it off of the `dcp_landuse` entity field `dcp_ceqrtype`. I want to leave this here since so far it's the only time we're using a ceqr type field in our forms. Maybe in the future if we end up incorporating another ceqr type field we can rearrange these optionsets? I don't want to get too far ahead of ourselves / try to implement something that's currently outside the scope of what we're doing.